### PR TITLE
Add check box to toggle plugin argument input

### PR DESCRIPTION
**Pull request by studentmain**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

- [x] [Searched](https://github.com/shadowsocks/shadowsocks-windows/search?q=is%3Apr&type=Issues) for similar pull requests
- [x] Compiled the code with Visual Studio

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New feature

---

### Description of your *pull request* and other information

插件参数默认隐藏以防止用户填错，如图所示
577b39b5-53b5-4dce-ad88-f3e851ffb66d

仅仅是demo，界面什么的还要调整一下

### CODE REVIEW

**Comment by celeron533** [comment]
感觉编辑服务器的界面在未来要重新设计一下了 🤔

**Comment by studentmain** [comment]
希望全局设置和逐服务器设置分开。便携模式，本地代理，前置代理，自定义PAC这些可以放在同一个窗口里面，以后还可以再塞些其他功能(例如全局模式自定义bypass列表)的输入框。

编辑服务器的话，问题主要集中在插件参数填错和程序没法处理空服务器上面。除了把添加删除这些换成图片，把输入框分组一下，我暂时没想到怎么设计。

现在是继续改改还是先关了等重新设计？

File: shadowsocks-csharp/Data/zh_CN.txt [hunk 1]
```diff
@@ -53,6 +53,7 @@ Show Password=显示密码
 Encryption=加密
 Plugin Program=插件程序
 Plugin Options=插件选项
+Use Plugin Arguement=需要命令行参数
```
Comment (chenshaoju) [inline shadowsocks-csharp/Data/zh_CN.txt]: It's "使用命令行参数" better?

Comment (studentmain) [reply shadowsocks-csharp/Data/zh_CN.txt]: Maybe "Require Plugin Arguement" 😅

Comment (wongsyrone) [reply shadowsocks-csharp/Data/zh_CN.txt]: typo: argument

File: shadowsocks-csharp/Data/zh_CN.txt [hunk 2]
```diff
@@ -53,7 +53,7 @@ Show Password=显示密码
 Encryption=加密
 Plugin Program=插件程序
 Plugin Options=插件选项
-Use Plugin Arguement=需要命令行参数
```
Comment (celeron533) [inline shadowsocks-csharp/Data/zh_CN.txt]: Typo:
Arguement --> Argument

**Comment by studentmain** [comment]
顺便一提，确认在VS2019RC下可以编译通过

**Pull request merged**